### PR TITLE
Update chatgpt-quick-actions

### PR DESCRIPTION
**Pull request by alanzchen**
Supports new models

### CODE REVIEW

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

File: extensions/chatgpt-quick-actions/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
 # ChatGPT Quick Actions Changelog
 
+## 2023-06-13
```
Comment (pernielsentikaer) [inline extensions/chatgpt-quick-actions/CHANGELOG.md:3]:
```suggestion
## [New models] - 2023-06-13
```

File: extensions/chatgpt-quick-actions/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+ - Adds support for the following models: `gpt-3.5-turbo-16k`, `gpt-4-32k-0613`.
+
+## 2023-06-03
```
Comment (pernielsentikaer) [inline extensions/chatgpt-quick-actions/CHANGELOG.md:7]:
```suggestion
## [Enhacements] - 2023-06-03
```

**Comment by pernielsentikaer** [comment]
@alanzchen do you mind checking if there are some merge conflicts, it seems like I can't merge it due to that.

**Comment by alanzchen** [comment]
@pernielsentikaer Done!

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**